dags/tpu_observability/utils/jobset_util.py

```diff
@@ -30,7 +30,7 @@
 from google.cloud.monitoring_v3 import types
 import kubernetes
 
-from dags.tpu_observability.utils import node_pool_util as node_pool
+from dags.tpu_observability.utils import node_pool_util
```

Comment:
maybe 
```suggestion
from dags.tpu_observability.utils.node_pool_util import Info as node_pool_info
```
so that the usage will be 
```
def xxx(node_pool: node_pool_info)
```